contracts/seed/Seed.sol

```diff
@@ -164,6 +168,7 @@ contract Seed {
         minimumReached  = false;
         maximumReached  = false;
         seedRemainder   = IERC20(_tokens[0]).balanceOf(address(this));
+        seedAmtAtStart  = IERC20(_tokens[0]).balanceOf(address(this));
```

Comment:
The problem is, at ```SeedFactory.deploySeed```, we tried to first pass the value. But while doing that, we were facing ```stack too deep``` issue because we were kind of exceeding the variable limit. So we are using balance of. And ```deploySeed``` is deploying seed, transferring the necessary seed tokens and immediately initialising the seed and then it emits event with the address of ```Seed``` contract. So I think it won't cause a big issue. As anyone won't get chance to transfer before ```SeedFactor.deploySeed```.

Reply:
I don't really understand what is the problem with just putting the params into an array.

But I also don't understand all the potential attack vectors.

Reply:
Hey @dkent600 , I have created an issue for this #152 , This PR have already a lot have changes. So if this is the only thing we need to improve in this PR, I can handle it in its own respective PR.

Reply:
I have solved #152 in this PR only.